python/lsst/ts/watcher/watcher_csc.py

```diff
@@ -274,7 +266,18 @@ async def output_alarm(self, alarm):
         )
 
     async def handle_summary_state(self):
-        self._enable_or_disable_model()
+        if self.summary_state == salobj.State.ENABLED:
+            if self.model is None:
+                raise RuntimeError("Bug! No model.")
```

Comment:
Considering removing the "Bug" part of the error message. Maybe something like

```
"Unable to initialize model. This is most likely due to a programatic error and might require restarting the component."
```

Reply:
I like Bug as a more efficient way to say "programming error", but I agree the message needs to be more helpful. I suggest this:

Bug: state is ENABLED but there is no model. Please restart the software and file a JIRA ticket.

Reply:
I just wonder what people will think when they see the error message. Your suggestion is fine.